bugzilla/base.py

```diff
@@ -1061,7 +1061,8 @@ def _supports_getbug_extra_fields(self):
 
 
     def _getbugs(self, idlist, permissive,
-            include_fields=None, exclude_fields=None, extra_fields=None):
+            include_fields=None, exclude_fields=None, extra_fields=None,
+            limit=None):
```

Comment:
It seems like with bugzilla.redhat.com, getbugs does not have a limit anymore (query() still does that). So i think this was a transient change?